PatternPal/PatternPal.Extension/ViewModels/StepByStepInstructionsViewModel.cs

```diff
@@ -17,6 +17,10 @@ public class StepByStepInstructionsViewModel : ViewModel
     {
         public ICommand NavigateHomeCommand { get; }
 
+        public string FilePath { get; set; }
+
+        public bool SBScontinue { get; }
```

Comment:
```suggestion
        public bool Continue { get; }
```
As you're already in a class specific to Step by Step, you don't need to include that in the property names here.

Reply:
This was replaced with an enum of modi.